mono-traversable/ChangeLog.md

```diff
@@ -1,5 +1,10 @@
 # ChangeLog for mono-traversable
 
+## 10.0.15.4
```

Comment:
It should bump to 1.0.16.0. The 10 looks like a typo, and since this adds a new instance we need to bump the minor version (the third component of the version number).